packages/platform-android/native_modules.gradle

```diff
@@ -160,7 +160,7 @@ class ReactNativeModules {
       }
       output = readBuffer.toString()
       if (!output) {
-        this.logger.error("${LOG_PREFIX}Unexpected empty result of running '${command}' command from '${directory}' directory.")
```

Comment:
This is `null` right now, so I have removed the extra logging for better error messages without "null" directory in the output.